### PR TITLE
Cleaning analyzer types

### PR DESCRIPTION
**Pull request by Chemaclass**
# Description

Introduced constants for every Symbol name (let, do, loop, etc) and clean some analyze symbol classes.

# Changes

- Included `Symbol::NAME_*` constants
- Used that constants instead of raw strings where does it makes sense
- Minor little cleaning
- No functionality was modified

### CODE REVIEW

File: src/php/Lang/Symbol.php [hunk 1]
```diff
@@ -8,6 +8,28 @@
...
+    public const NAME_IF = 'if';
+    public const NAME_APPLY = 'apply';
+    public const NAME_LET = 'let';
+    public const NAME_PHP_NEW = 'php/new';
+    public const NAME_PHP_OBJECT_CALL = 'php/->';
+    public const NAME_PHP_OBJECT_STATIC_CALL = 'php/::';
+    public const NAME_PHP_ARRAY_GET = 'php/aget';
+    public const NAME_PHP_ARRAY_SET = 'php/aset';
+    public const NAME_PHP_ARRAY_PUSH = 'php/apush';
+    public const NAME_PHP_ARRAY_UNSET = 'php/aunset';
```
Comment (Chemaclass) [inline src/php/Lang/Symbol.php:25]: @jenshaase I was thinking between `NAME_PHP_AGET` and `NAME_PHP_ARRAY_GET`. I think the first one is closer to the real symbol itself, but the second one is more expressive, but what do you think?

File: src/php/Reader.php [hunk 2]
```diff
@@ -129,7 +129,7 @@ public function readExpression(Generator $tokenStream)
                     throw $this->buildReaderException('Unterminated list');
 
                 case Token::T_QUOTE:
-                    return $this->readWrap($tokenStream, 'quote');
+                    return $this->readWrap($tokenStream, Symbol::NAME_QUOTE);
```
Comment (Chemaclass) [inline src/php/Reader.php:132]: @jenshaase  I'm not sure about this one 🤔 because what's about ''unquote'? Should we treat it as a symbol as well?

Comment (jenshaase) [reply src/php/Reader.php:132]: Yes, `unquote`, `quote` and `unquote-splicing` are all Symbol names. They are mostly used in the Quasiqoute class.

File: src/php/Lang/Symbol.php (hunk 1 above)
Comment (jenshaase) [reply src/php/Lang/Symbol.php:25]: `NAME_PHP_ARRAY_GET` is fine for me

File: src/php/Reader.php (hunk 2 above)
Comment (Chemaclass) [reply src/php/Reader.php:132]: Ok, then I will create those constants as well :)

**Pull request merged**